pydis_site/apps/content/resources/guides/python-guides/discord-messages-with-colors.md

```diff
@@ -61,8 +66,14 @@ Result:
 
 ![Background and text color result](/static/images/content/discord_colored_messages/result.png)
 
+### ANSI Colors Showcase
+
 The way the colors look like on Discord is shown in the image below:
 
 ![ANSI Colors](/static/images/content/discord_colored_messages/ansi-colors.png)
 
-Note: If the change as not been brought to you yet, or other users, then you can use other code blocks in the meantime to get colored text. See **[this gist](https://gist.github.com/matthewzring/9f7bbfd102003963f9be7dbcf7d40e51)**.
+*Message sent to get the output of above can be found [here](https://gist.github.com/kkrypt0nn/a02506f3712ff2d1c8ca7c9e0aed7c06#file-ansi-colors-showcase-md).*
+
+#### Disclaimer
+
+***Note**: If the change as not been brought to you yet, or other users, then you can use other code blocks in the meantime to get colored text. See this gist: <https://gist.github.com/matthewzring/9f7bbfd102003963f9be7dbcf7d40e51>*
```

Comment:
Does this only apply to mobile users? If so, should it be more specific e.g. "Note that this feature doesn't work on mobile yet"

Reply:
I heard from some users in the past few weeks they don't see the ANSI colors on their Linux/Windows & Stable client - will confirm that next week and update accordingly

Reply:
Seems like they were using an outdated client version - I've tested around and seen the change now in stable everywhere (besides mobile).

Updated the disclaimer accordingly as well

Reply:
Thanks for pointing that out :) - has been done